src/components/Editor/components/Main.js

```diff
@@ -17,6 +18,8 @@ class Main extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      paneStyle: { transition: "none" },
```

Comment:
Don't think this style affects anything. When I ctrl-f there's only one instance of it. There was an earlier change that removed a lot of the inline styling so it was probably a mis-merge